Bottom: 052e7162bf180323e179c59bb1fc8038d8ba4654
Top:    7bc13eeb89f93d4e8c5a0ba2448cd290f489af55
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 16:11:39 +0100

Refresh of hid-gtk-use-gtk_notebook_get_n

---

diff --git a/src/hid/gtk/gui-config.c b/src/hid/gtk/gui-config.c
index 851571a..698f57a 100644
--- a/src/hid/gtk/gui-config.c
+++ b/src/hid/gtk/gui-config.c
@@ -1980,7 +1980,7 @@ config_page_create (GtkTreeStore * tree, GtkTreeIter * iter,
 
   vbox = gtk_vbox_new (FALSE, 0);
   gtk_notebook_append_page (notebook, vbox, NULL);
-  page = g_list_length (notebook->children) - 1;
+  page = gtk_notebook_get_n_pages (notebook) - 1;
   gtk_tree_store_set (tree, iter, CONFIG_PAGE_COLUMN, page, -1);
   return vbox;
 }
